src/pages/signIn/signUp.jsx

```diff
@@ -0,0 +1,83 @@
+import { Formik } from 'formik';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { getUserInfo } from '../../redux/actions/actionCreators';
+import { initValuesForSignUp, ValidSchemeForSignUp } from './validation';
+import { SignUpForm, SignForm, SignInButton, StyledLink, StyledTitle } from './styled';
+import ValidInput from './validInput';
+import { useMutationDataServer } from '../../hooks/mutation/dataServerMutation';
+
+const SignUp = () => {
+    const dispatch = useDispatch();
+
+    const mutation = useMutationDataServer();
+
+    const loadDataToServer = (userCredentials) => {
```

Comment:
loadDataToServer, omg, how many times I should say to rename it to understand what the function does

Reply:
You even don't need the function at all.
just create 1 function for form submitting:
const submitForm = async(values)=>{
dispatch(getUserInfo(values));
        mutation.mutate(values);
        }
 and pass it in Formik onSubmit